src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -217,6 +217,23 @@ export function ChangeDataView({
         ) : (
           <React.Fragment />
         ),
+        <EuiContextMenuItem
+          key="refresh"
+          icon="refresh"
+          data-test-subj="data-view-refresh-fields"
+          onClick={async () => {
+            const dataView = await dataViews.get(currentDataViewId!);
```

Comment:
@mattkime this would hit the dataView cache right? So if we want to have a data view reload (so also runtime fields would be reloaded), this function would need to trigger a fetch of the saved object and no a cache entry, right?

Reply:
@kertal Yes, that will hit the cache